vendor/brave-ios/BATBraveRewards.h

```diff
@@ -94,8 +94,7 @@ NS_SWIFT_NAME(BraveRewards)
 /// Report that a page has loaded in the current browser tab, and the HTML is available for analysis
 ///
 /// @note Send nil for `adsInnerText` if the load happened due to tabs restoring
-///       after app launch or if response header for the page load contains
-///       "cache-control: no-store"
+///       after app launch
```

Comment:
@kylehickinson I raised an issue for brave-ios so remove the cache control business logic from the iOS app (comment above updated to reflect the needed change)